CODEOWNERS

```
@@ -0,0 +1,88 @@
+# Code Owners
+
+# Scripts
+/scripts/ @ndelangen @kasperpeulen
+
+# Addons
+/code/addons/a11y/                 @ndelangen @yannbf
+/code/addons/actions/              @ndelangen @yannbf
+/code/addons/backgrounds/          @yannbf @ndelangen
+/code/addons/controls/             @kasperpeulen @ndelangen
+/code/addons/docs/                 @kylegach @kasperpeulen
+/code/addons/essentials/           @valentinpalkovic @ndelangen
+/code/addons/gfm/                  @ndelangen @valentinpalkovic
+/code/addons/highlight/            @yannbf @valentinpalkovic
+/code/addons/interactions/         @yannbf @ndelangen
+/code/addons/jest/                 @ndelangen
+/code/addons/links/                @yannbf @kylegach
+/code/addons/measure/              @yannbf @valentinpalkovic
+/code/addons/outline/              @yannbf @valentinpalkovic
+/code/addons/storyshots-core/      @ndelangen
+/code/addons/storyshots-puppeteer/ @ndelangen
+/code/addons/storysource/          @ndelangen
+/code/addons/toolbars/             @ndelangen @kylegach
+/code/addons/viewport/             @yannbf @ndelangen
+
+# Builder
+/code/builders/builder-manager/  @ndelangen @valentinpalkovic
+/code/builders/builder-vite/     @ndelangen @valentinpalkovic @IanVS
```

```suggestion
/code/builders/builder-vite/     @JReinhold @valentinpalkovic @IanVS
```

Based on https://chromaticqa.slack.com/archives/C04S3F04Q68/p1687909632587579?thread_ts=1687877773.854549&cid=C04S3F04Q68